syncstorage/storage/sql/dbconnect.py

```diff
@@ -363,6 +379,12 @@ def __init__(self, sqluri, create_tables=False, pool_size=100,
             collections.create(self.engine, checkfirst=True)
             user_collections.create(self.engine, checkfirst=True)
             batch_uploads.create(self.engine, checkfirst=True)
+            migration.create(self.engine, checkfirst=True)
+            # add this row for testing:
```

Comment:
It seems unusual to have the production code add values specifically for testing, why do we need to do this here rather than e.g. in the test setup code?

Reply:
(Actually, having gotten through the rest of the PR, I don't see where this value is used in any tests - maybe it's a leftover from manual testing?)

Reply:
Yeah, it's left over from a previous test that was pulled. Thanks!